_posts/2025-08-10-update.md

```diff
@@ -18,6 +18,11 @@ slug: 2025-08-10-update
 
 ## KEP of the Week
 
+[KEP 5080: Ordered Namespace Deletion](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/5080-ordered-namespace-deletion/README.md)
+
+This KEP introduces a deterministic, security-aware ordered deletion of all resources within a namespace. Previously, namespace deletion could remove resources in a non-deterministic order that could lead to awkward or risky gaps. When deleting a namespace with the `OrderedNamespaceDeletion` Feature Flag enabled, Kubernetes tears down namespace objects in waves, so that Pods go first and the crucial resources like`NetworkPolicy` Don’t disappear while Pods are still running
```

Comment:
```suggestion
This KEP introduces a deterministic, security-aware ordered deletion of all resources within a namespace. Previously, namespace deletion could remove resources in a non-deterministic order that could lead to awkward or risky gaps. When deleting a namespace with the `OrderedNamespaceDeletion` Feature Flag enabled, Kubernetes tears down namespace objects in waves, so that Pods go first and the crucial resources like`NetworkPolicy` don't disappear while Pods are still running.
```

Just some punctuation fixes